### PR TITLE
rename various options for consistency reasons

### PR DESCRIPTION
**Pull request by Dudemanguy**
This better reflects what it actually does. As a bonus, script writers won't be misled into thinking that fps displays the actual video or display fps.

### CODE REVIEW

File: filters/f_decoder_wrapper.c [hunk 1]
```diff
@@ -113,7 +113,7 @@ static int decoder_list_help(struct mp_log *log, const m_option_t *opt,
 const struct m_sub_options dec_wrapper_conf = {
     .opts = (const struct m_option[]){
         {"correct-pts", OPT_BOOL(correct_pts)},
-        {"fps", OPT_DOUBLE(force_fps), M_RANGE(0, DBL_MAX)},
+        {"force-container-fps", OPT_DOUBLE(force_fps), M_RANGE(0, DBL_MAX)},
```
Comment (kasper93) [inline filters/f_decoder_wrapper.c:116]: Maybe `--override-container-fps` to be in parity with `--override-display-fps`

Comment (Dudemanguy) [reply filters/f_decoder_wrapper.c:116]: Ah yes, consistency is good.

Comment (Dudemanguy) [reply filters/f_decoder_wrapper.c]: Maybe a few other minor adjustments too (`force_fps` -> `override_fps`).

**Comment by Dudemanguy** [comment]
So @guidocella pointed out some inconsistencies in the option naming in IRC, so I went ahead and included additional commits to change those. Feel free to to comment.

**Comment by guidocella** [comment]
We should pluralize `--sub-ass-style-override` to `--sub-ass-style-overrides` because the other list options that aren't abbreviated are plural, looks good otherwise. Is `hwdec-current` more problematic to rename because there is no `OPT_REPLACED` mechanism?

**Comment by Dudemanguy** [comment]
>We should pluralize --sub-ass-style-override to --sub-ass-style-overrides

Fair point.

>Is hwdec-current more problematic to rename because there is no OPT_REPLACED mechanism?

Nah, it's more that I wasn't sure. There's already a `hwdec-interop` property for example so it creates a different inconsistency if you change it to `current-hwdec`.

**Comment by guidocella** [comment]
IMHO it's not an inconsistency because "hardware decoding interop driver" is correct English while "hardware decoding current" is just nonsense.

**Comment by Dudemanguy** [comment]
That's true, but it's weird imo to have the property names not start the same. Like most video-related properties are `video-something`.

**Comment by N-R-K** [comment]
While we're at it, can we find a better name for `demuxer-hysteresis-secs` ?

**Comment by Dudemanguy** [comment]
Seems surprisingly not controversial? I thought people were going to get upset at me for changing `--screenshot-directory`.

>While we're at it, can we find a better name for demuxer-hysteresis-secs ?

Name seems OK to me? Haven't actually looked at you PR yet though.

**Comment by N-R-K** [comment]
>Name seems OK to me?

I was hoping we can use something that's in people's everyday vocabulary. Something like `demuxer-sleep-until` or `demuxer-wake-when-remaining` etc (better suggestions welcome).

**Pull request merged**